GitHub Is An Arse

```diff
@@ -0,0 +1,35 @@
+// Dylan Tkach
+// CPT-263-27
+// Assignment 3 - Loops
+
+package assignment.pkg3.loops;
+
+import java.util.Scanner;
+
+public class Assignment3Loops {
+    
+    public static void main(String[] args) {
+        
+        
+        Scanner keyboard = new Scanner(System.in);
+        print("Please enter a word: ");
+        String word = keyboard.next();
+        
+        while(!word.equalsIgnoreCase("EXIT")){
+        String reversedWord = "";
+        for(int x = word.length() - 1; x >= 0; x--){
+            char value = word.charAt(x);
+            reversedWord += String.valueOf(value);
+            }
+        
```

Comment:
I like that you used the valueOf method as suggested in the assignment details. I started out by declaring my variables as strings instead of characters and then found out after that that I did not need to use that method to print my word, but I still like to see it in action :) I also liked that you incorporated the equalsIgnoreCase method.